main.tf

```diff
@@ -76,6 +76,14 @@ resource "google_healthcare_fhir_store" "fhir_stores" {
     }
   }
 
+  dynamic "notification_configs" {
```

Comment:
Is the above `notification_config` kept for backwards compatibility purpose?

Reply:
Yes, the API did not remove that field, thus keeping that field here too.